WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderPostAdapter.java

```diff
@@ -1302,4 +1308,8 @@ protected void onPostExecute(Boolean result) {
             mIsTaskRunning = false;
         }
     }
+
+    private String getUrlWithoutProtocol(String url) {
+        return url.replaceFirst("^(http[s]?://www\\.|http[s]?://|www\\.)", "");
```

Comment:
@ashiagr - I might consider writing a unit test for this one if you haven't already.

Reply:
I think we could reuse UrlUtils.removeScheme(...) for this. Wdyt?

Reply:
Reused here: 68c5274